client/src/sendMessageArea.tsx

```diff
@@ -121,14 +125,18 @@ export const SendMessageArea = ({
           }
         `}
         onClick={() => sendItem()}
-        disabled={!message && !payloadToBeSent}
+        disabled={(!message && !payloadToBeSent) || isItemSending}
```

Comment:
This expression is starting to get a bit complicated.  Not sure if:
`!(message || payloadToBeSent) || isItemSending`
is any improved though.

Reply:
fair, I've improved it slightly in https://github.com/guardian/pinboard/pull/158/commits/938eb3d3cb20182fcf2afe8194da8f5bd458d8a6

Reply:
Still not completely sure it is better. But it isn't worse 🤷